### PR TITLE
fix(fma): continuation of presel text fix

### LINKED ISSUE #8217: PFD SPEED SEL message
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-09-16T06:20:39+00:00",
    "ref": "refs/heads/master",
    "sha": "4783ebcadace8462bc71af4f8c1bb4c1ebdb322c",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:4783ebca",
    "version": "v0.11.0-dev.4783ebc"
}
```

### Describe the bug

The SPEED SEL message in the line 3 of the FMA it's missing the ":".
It displays "SPEED SEL 225"

### Expected behavior

Should display "SPEED SEL: 225"

### Steps to reproduce

1. Prior to takeoff, presel a speed in PERF CLB
2. During climb check the 3rd line of the FMA

### References (optional)

DSC-22_30-80-20
image

https://youtu.be/RhyQK7Jz56g?t=94

### Additional info (optional)

This was tested with a preselected CLB speed, but should be the same for preselected CRZ speed

### Discord Username (optional)

Bernardo R

### PR DESCRIPTION
**Pull request by BravoMike99**
Fixes #8217 

## Summary of Changes

This PR adds ":" to the SPEED SEL and MACH SEL text and slightly adjusts their position(continuation of #8230). 
## Screenshots (if necessary)

Left side is after while right side is before.
MACH SEL
SPEED SEL

## References

Provided in #8230

## Additional context

I had to map the "|" character to a bottom ":" character in the font file considering it is shared between multiple displays. Since the preselect ":" is aligned to the bottom, using the existing ":" would not work.

Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

Preselect a speed in the PERF CLB page (before takeoff). Make sure speed sel shows on the FMA with the ":" (if A/B is armed, it will only show after A/B message disappears).
Preselect a mach speed in the PERF CRZ page. Once climb phase is active make sure mach sel shows on the FMA with the ":".

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/instruments/src/PFD/FMA.tsx [hunk 1]
```diff
@@ -630,18 +630,31 @@ class AB3Cell extends DisplayComponent<CellProps> {
 
     private textSub = Subject.create('');
 
+    private text2Sub = Subject.create('');
+
+    private textXPosSub= Subject.create(0);
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/PFD/FMA.tsx:635]:
```suggestion
    private textXPosSub = Subject.create(0);
```

**Review by tracernz — APPROVED** [review]
Just a small formatting fix.

**Comment by lukecologne** [comment]
Hm, not sure if we need a new character in the font for this. From the refs for example [here](https://www.youtube.com/watch?v=n3TJMro_NzA&t=523s) it looks like it could simply be a smaller font size compared to the rest?

**Comment by tracernz** [comment]
The vertical alignment of the : is very different to what's currently in the font.

**Comment by BravoMike99** [comment]
I made the split into two different text fields just to manipulate the spacing between the presel text and preselected value since it was one of the suggestions of improvement in the other PR and seems to differ IRL between the MACH SEL and SPEED SEL. This is how the spacing looks on a single text element (ignore the : , used master branch).
MACH SEL2
SPEEDSEL

**Comment by lukecologne** [comment]
Is the spacing really different between SPEED and MACH SEL? I assume you used the pictures from shomas from the original PR
(Pictures here side by side for convenience)

| Header | Header |
|--------|--------|
271003339-ab61707b-7edd-4e58-bdce-ed2e86ce26d9 | 271796232-d25e193d-c6ec-4ec8-974f-ad6dd0921bee 

The space in the MACH SEL is very obvious of course, but I'm pretty sure I see it in the SPEED SEL too (you have to compare either both the leftmost or rightmost edges of the blur).

You're right about the colon, it's weird that the alignment is so different... Although if you're splitting the text fields anyways it might be easier to just add a third text field just for the colon, and move that downwards, instead of doing a whole new character in the font

**Comment by BravoMike99** [comment]
Yeah I used shomas pics and maybe I wasn't clear enough, i meant the space after the ":" which seems to be smaller in the SPEED SEL case. Regarding the colon, I can do that if that's preferable.

**Comment by lukecologne** [comment]
I don't know if that's preferable, just giving alternatives

**Comment by Popespice** [comment]
This appears to be fixed, resolving both this ticket and #8230. Please close and PR

**Pull request merged**